about.html

```diff
@@ -11,54 +11,67 @@
 
 </head>
 
+<header>
+    <div class="about-page">
+    <section Class="nav-bar">
+            <a href="index.html" class="logo"><img src="img/lambda-black.png" alt="Lambda School Logo"></a>
+    
+     <a href="">Home</a>     
+     <a href="about.html" class="link">About</a>
+     <a href="">Products</a>  
+     <a href="">Blog</a>   
+     <a href="">Contact</a>  
+     </nav>
```

Comment:
Here is the end of your </nav>, but where is the beginning? I think you meant o have this start with the <section> on line 16 that has a class of "nav-bar". Be sure to consistently use either <section> or <nav>. I think these were supposed to correspond but just quit didn't.